### ISSUE TITLE
AssettoCorsaSharedMemory.dll

### ISSUE THREAD

**Issue by ikkennigij121** [mdjarv/assettocorsasharedmemory#13]
Hello,

I'm pretty new to programming and trying to figure out how this works I struggle in locating AssettoCorsaSharedMemory.dll to add it as a reference.

Any idea where I can find this? should this be included in the AC installation or in this project?

**Comment by cv256** [comment]
check this project (.net+ this library + arduino + android) it's my moving car rig:
https://github.com/cv256/CVJoy

**Comment by ikkennigij121** [comment]
> check this project (.net+ this library + arduino + android) it's my moving car rig:
> https://github.com/cv256/CVJoy

Thank you for your reply but not really what I was looking for. 

My issue was that I took a simple example to understand how it would work ( https://github.com/Thomseeen/assetto-corsa-telemetry-router) but I was missing the reference to the dll.

I found out that it was as simple as downloading the assetocorsasharedmemory library, which you also mentioned. (https://github.com/mdjarv/assettocorsasharedmemory) and in the telemetry project, just reference the dll found in the folder from the assetocorsasharedmemory library (\assettocorsasharedmemory-master\bin\Release)
